apps/approval/tests.py

```diff
@@ -97,6 +99,7 @@ def setUp(self):
         self.logger = logging.getLogger(__name__)
 
     def testEmailWhenMembershipDenied(self):
```

Comment:
I see this is not something you have edited in this PR, but aren't most test in OW4 underscored and not camel-cased?

Reply:
You are correct. I think some of the oldest parts of this project has som camelCased parts, but they shouldn't be

Reply:
Would you care to fix it, or should we just leave it untouched and add it to the backlog?

Reply:
I think its nicer to just have a cleanup of all those by themselves

Reply:
Want ahead and created a PR for it #2477